packages/roosterjs-editor-dom/lib/utils/getTableOriginalColor.ts

```diff
@@ -0,0 +1,13 @@
+import { TableMetadata } from '../table/tableMetadata';
+
+const SELECTION_COLOR_OPACITY = TableMetadata.SELECTION_COLOR_OPACITY;
+
+/**
+ * @internal
+ * Retrieve the color to be applied when a cell is selected
+ * @param colorString Color
+ * @returns color to be applied when a cell is selected
+ */
+export function getHighlightColor() {
```

Comment:
move this function to VTable since it is not used in other places, and no need to export